### PR TITLE
activity_create

### PR DESCRIPTION
**Pull request by hienbx94**
I have created pull request. Please check it.

### CODE REVIEW

**Comment by ttuan** [comment]
OK

**Comment by trantube94** [comment]
OK

File: app/models/user_task.rb [hunk 1]
```diff
@@ -2,12 +2,19 @@ class UserTask < ActiveRecord::Base
...
 
   before_create :init_relation
+  after_create :init_activity 
 
   def init_relation
     self.user_id = self.user_subject.user_id
     self.subject_id = self.user_subject.subject_id
     self.course_id = self.user_subject.course_id
   end
+
```
Comment (ThuBM) [inline app/models/user_task.rb]: move init_relation and init_activity to private

File: app/models/user_task.rb [hunk 2]
```diff
@@ -2,12 +2,19 @@ class UserTask < ActiveRecord::Base
...
+  def init_activity
+    ApplicationController.helpers.create_activity self.user_id,
```
Comment (ThuBM) [inline app/models/user_task.rb]: không sử dụng helper trong model

**Pull request merged**